src/main/java/ay2122s1_cs2103t_w16_2/btbb/storage/UserPrefsStorage.java

```diff
@@ -1,15 +1,15 @@
-package seedu.address.storage;
+package ay2122s1_cs2103t_w16_2.btbb.storage;
 
 import java.io.IOException;
 import java.nio.file.Path;
 import java.util.Optional;
 
-import seedu.address.exception.DataConversionException;
-import seedu.address.model.ReadOnlyUserPrefs;
-import seedu.address.model.UserPrefs;
+import ay2122s1_cs2103t_w16_2.btbb.exception.DataConversionException;
+import ay2122s1_cs2103t_w16_2.btbb.model.ReadOnlyUserPrefs;
+import ay2122s1_cs2103t_w16_2.btbb.model.UserPrefs;
 
 /**
- * Represents a storage for {@link seedu.address.model.UserPrefs}.
+ * Represents a storage for {@link UserPrefs}.
```

Comment:
ay2122s1_cs2103t_w16_2.btbb wasn't added back
```suggestion
 * Represents a storage for {@link ay2122s1_cs2103t_w16_2.btbb.model.UserPrefs}.
```